frontend/app/utils/application-code-utils.ts

```diff
@@ -55,3 +57,15 @@ export function formatSubmissionApplicationCode(applicationCode: string): string
   if (!/^\d{13}$/.test(strippedCode)) return applicationCode;
   return (strippedCode.match(/....$|.../g) ?? []).join(' ');
 }
+
+/**
+ *
+ * @param statusId - the statusId of the application to check
+ * @returns the "info" | "success" | "danger" which is consumed by the ContextualAlert component
+ */
+export function getContextualAlertType(statusId: string | undefined) {
```

Comment:
If `getStatusIdWithSin` and/or `getStatusIdWithoutSin` do not find a statusId, does it return undefined? or null? 

If null, we might want to do this: 

`getContextualAlertType(statusId: string | undefined | null)`

Reply:
good point.  I think it's probably better to operate on `statusId` (which will always be a string or null) instead of the `clientFriendlyStatus?.id` that I was before.

Reply:
resolving